dockers/docker-snmp/snmpd.conf.j2

```diff
@@ -32,16 +32,41 @@ agentAddress udp6:161
                                                  #  system + hrSystem groups only
 view   systemonly  included   .1.3.6.1.2.1.1
 view   systemonly  included   .1.3.6.1.2.1.25.1
-
                                                  #  Default access to basic system info
-{% if snmp_rocommunities %}
-{% for community in snmp_rocommunities %}
+
+{% if SNMP_COMMUNITY is defined %}
+{% for community in SNMP_COMMUNITY %}
+{% if SNMP_COMMUNITY[community]['TYPE'] == 'RO' %}
 rocommunity {{ community }}
 rocommunity6 {{ community }}
+{% else %}
```

Comment:
Empty else can be removed. Is it placed for a specific reason?

Reply:
fixed. Thanks!